### DISCUSSION TITLE
legend position: right 个数怎么限制  🤔 [QUESTION]

### DISCUSSION THREAD

**Discussion by TopAlien** [antvis/G2Plot#3566] [Q&A]
### 🐛 Question description [Please make everyone to understand it]
我想限制6每页个，无果，该怎么设置。
### 💻 Link to minimal reproduction
https://codesandbox.io/s/womt8q?file=/index.ts
Please provide a link by forking these links [g2plot](https://codesandbox.io/s/g2plot-reproduction-template-gyej1) or GitHub repo. What is a minimal reproduction, and why is it required?

### 🚑 Any additional [like screenshots]

* **G2Plot Version**:
* **Platform**:

**Comment by TopAlien** [comment]
<img width="538" alt="image" src="https://user-images.githubusercontent.com/33052129/231628553-c2e7daca-3c2e-445a-9ccc-cc618f92692e.png">

**Comment by controllerz** [comment]
![image](https://user-images.githubusercontent.com/22693880/231686798-abbd0da3-882b-406a-b729-e173b810ea97.png)
总高度 和 单独高度是6的倍数就能实现吧？

**Comment by TopAlien** [comment]
> ![图像](https://user-images.githubusercontent.com/22693880/231686798-abbd0da3-882b-406a-b729-e173b810ea97.png) 总高和单一高是6的倍数就能现实吧？

不行🙅，如果6条的话就没办法设置出来 只展示5条，需要7条及以上的数据才能正确的分页。 但是又必须限制为5条

**Reply by hustcc** [reply]
你需要只展示五条？超出的都不展示？

**Reply by TopAlien** [reply]
每页5条，多的分页展示。现在碰到的特殊情况是，当图例只有6个时，分页5个没用。
